HISTORY.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Fixed issue with GS credentials, using default auth enables a wider set of authentication methods in GS
```

Comment:
@ljyanesm Could you also add the issue, PR, and your username for credit (if you want)